app/models/group.rb

```diff
@@ -0,0 +1,5 @@
+class Group < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  has_many :group_entities, dependent: :destroy
+  has_many :entities, through: :group_entities, dependent: :destroy
```

Comment:
- [ ] OPTIONAL: I think validations are safer on the model side than the view side. It is good you have added validations in the form inputs but there might be a possibility that scammers make requests without form inputs. :pray: